duneanaobj/StandardRecord/SRRecoParticle.h

```diff
@@ -0,0 +1,38 @@
+////////////////////////////////////////////////////////////////////////
+/// \file    SRRecoParticle.h
+/// \brief   information for a reco particle
+/// \author  L. Di Noto
+/// \date    Jan. 2023
+////////////////////////////////////////////////////////////////////////
+
+#ifndef DUNEANAOBJ_SRRECOPARTICLE_H
+#define DUNEANAOBJ_SRRECOPARTICLE_H
+
+#include "duneanaobj/StandardRecord/SRLorentzVector.h"
+#include "duneanaobj/StandardRecord/SRVector3D.h"
+
+namespace caf
+{
+  class SRRecoParticle
+  {
+    public:
+
+      int primary=0;              //1 if primary, 0 otherwise
+      int   reco_trkid     = 0;   //trackid of reco particle
```

Comment:
I'm not sure I understand what this variable is used for.  What "track ID" is this?  Is it meant to be the index of an associated `SRTrack` object?  If so, this is a bit complicated, because where that `SRTrack` lives isn't fixed (a ND-LAr or TMS `SRRecoParticle` won't have the same set of `SRTrack`s as SAND).  I think we probably want to find a different way to encode whatever information this is, but I need to understand more from you on what you're trying to do. 😄

Reply:
We discussed this.  For now, we're going to promote more "physics variables" into the `SRRecoParticle` (i.e.: reco vertex location & time, reco momentum), and that may be enough.  Lea's also going to look into an `SRSANDTrkAssn` (analogous to `SRNDTrackAssn`) which would allow the association of track fragments across SAND subdetectors (and would live inside `SRSAND`).

How to associate a particle to its "source" object (whether it be track or shower or whatever)---and whether we actually need to do that---is a complicated problem that we'll have to consider in the larger redesign (#12), but we aren't going to try to solve it in this PR.